Signal/translations/en.lproj/Localizable.strings

```diff
@@ -1417,6 +1417,9 @@
 /* action sheet item */
 "SHARE_ACTION_TWEET" = "Twitter";
 
+/* alert title */
+"SHARE_PROFILE_FAILED_TITLE" = "Failed to Share Your Profile";
```

Comment:
I wonder if we should bother the user with this.  We send our profile key with every message, right?  So the other user will see your profile as soon as you have any contact with them. "Don't make me think."

Reply:
To be clear, are you suggesting we get rid of the retry logic as well as the alert?